### PR TITLE
Implement travis

### PR DESCRIPTION
**Pull request by koggs**
#### What I did:
-Add travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: ruby
+rvm:
+  - 2.5.2
```
Comment (dgmstuart) [inline .travis.yml]: Do we also need postgres in order to run the tests?
https://docs.travis-ci.com/user/database-setup/#postgresql

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# This file is auto-generated from the current state of the database. Instead
```
Comment (dgmstuart) [inline db/schema.rb]: Is this related to Travis? If not is it worth removing it from this PR (eg. using `git commit --amend`) and commit it separately?

**Review by dgmstuart — APPROVED** [review]
Looks like a valid Travis file, but I think it maybe needs a bit more config in order to run tests

File: .travis.yml (hunk 1 above)
Comment (koggs) [reply .travis.yml]: you are right. I will add that :)

File: db/schema.rb (hunk 2 above)
Comment (koggs) [reply db/schema.rb]: it is done :)

**Review by juliaramstedt — APPROVED** [review]
